### PR TITLE
Import kube package(cue) by GroupVersion

### PR DESCRIPTION
**Pull request by leejanee**
Resources of the same kind may have different groups and versions in kubernetes

This PR extend [pr](https://github.com/oam-dev/kubevela/pull/1211) and support import kube package(cue) by GroupVersion

usage:
```
apiVersion: core.oam.dev/v1alpha2
kind: WorkloadDefinition
metadata:
  name: worker-import
  namespace: vela-system
  annotations:
    definition.oam.dev/description: "Long-running scalable backend worker without network endpoint"
spec:
  definitionRef:
    name: deployments.apps
  extension:
    template: |
      import "k8s.io/apps/v1"
      output: v1.#Deployment & {
          metadata: {
              annotations: {
                  if context["config"] != _|_ {
                      for _, v in context.config {
                          "\(v.name)" : v.value
                      }
                  }
              }
          }
          spec: {
              selector: matchLabels: {
                  "app.oam.dev/component": context.name
              }
              template: {
                  metadata: labels: {
                      "app.oam.dev/component": context.name
                  }

                  spec: {
                      containers: [{
                          name:  context.name
                          image: parameter.image

                          if parameter["cmd"] != _|_ {
                              command: parameter.cmd
                          }
                      }]
                  }
              }

              selector:
                  matchLabels:
                      "app.oam.dev/component": context.name
          }
      }

      parameter: {
          // +usage=Which image would you like to use for your service
          // +short=i
          image: string

          cmd?: [...string]
      }
```

### CODE REVIEW

**Comment by wonderflow** [comment]
rer #1195

**Review by hongchaodeng — APPROVED** [review]
lgtm

**Comment by hongchaodeng** [comment]
Test failed @leejanee

File: pkg/dsl/definition/package.go [hunk 1]
```diff
@@ -120,64 +143,83 @@ func (pd *PackageDiscover) addKubeCUEPackagesFromCluster(apiSchema string) error
...
-		pkgName := genPackageName(v.Group, v.Version)
-		pkg, ok := packages[pkgName]
-		if !ok {
-			pkg = newPackage(pkgName)
-		}
+}`, v.Kind, k, v.Kind, apiVersion)
 
-		mykinds := groupKinds[pkgName]
-		mykinds = append(mykinds, v.Kind)
+		pkgBuild := func(dgvr domainGroupVersionKind, pkgName, fname string) error {
```
Comment (hongchaodeng) [inline pkg/dsl/definition/package.go:178]: Make it a function?

**Review by hongchaodeng — APPROVED** [review]
lgtm

**Review by wonderflow — APPROVED** [review]

**Pull request merged**